build/configs/artik055s/audio/defconfig

```diff
@@ -1231,4 +1249,4 @@ CONFIG_ENABLE_UPTIME=y
 
 #
 # Things Management
-#
+#
```

Comment:
please check this.

Reply:
Just deleted teh newline charatcter after the hash. Its okay & should n't be a problem.